frontend/storybook/src/components/entry/JIG/JIG-panel.ts

```diff
@@ -0,0 +1,145 @@
+ import "@elements/entry/jig/publish/panel/jiggling-body";
+import "@elements/entry/jig/publish/panel/side-bar-number";
+import "@elements/entry/home/TOSORT/column-list";
+import "@elements/entry/jig/publish/panel/jig-squad";
+import "@elements/entry/jig/publish/panel/jig-panel";
+
+ 
+
+   
+
+export default {
+  title: 'Homepage',
+}
+
```

Comment:
several of the below items look like they are hardcoded and can be moved into elements (the svgs especially)